docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -51,7 +54,45 @@ function su_humsci_profile_install() {
 /**
  * Enables config readonly module after profile finishes.
  */
-function su_humsci_profile_lock_config(&$install_state){
+function su_humsci_profile_lock_config(&$install_state) {
   install_finished($install_state);
   \Drupal::service('module_installer')->install(['hs_config_readonly']);
 }
+
+/**
+ * Create config split entities.
+ */
+function su_humsci_profile_update_8002() {
+  \Drupal::service('module_installer')->install([
+    'config_split',
+    'config_ignore',
+  ]);
+
+  // Install all the default configs first which includes environment specific
+  // config splits.
+  $source = new FileStorage('../config/default');
```

Comment:
This is handy. Setting up config splits is a pain. Perhaps a drush function one day.